daemon/cluster/executor/backend.go

```diff
@@ -61,6 +61,7 @@ type Backend interface {
 	PluginManager() *plugin.Manager
 	PluginGetter() *plugin.Store
 	GetAttachmentStore() *networkSettings.AttachmentStore
+	HasExperimental() bool
```

Comment:
I'm a bit worried that this will open the gate to adding `HasXX` for more options, and we end up with a giant interface.

I was planning to do a PR to make `SystemInfo` more granular; would that work?

Reply:
What's the worry here? I'd prefer to have the interfaces we need rather than a generic do-everything interface.

Reply:
Yeah, that's fair. I was just a bit concerned that every option from systemInfo would get to be added to the interface (if we need to know if specific _features_ are available, it could be a `HasFeature("foo")`, to allow more granular detection)